azurerm/resource_arm_container_registry.go

```diff
@@ -57,6 +57,20 @@ func resourceArmContainerRegistry() *schema.Resource {
 				Default:  false,
 			},
 
+			"georeplication_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
```

Comment:
Is this needed?  Could we calculate this on the fly based on the `georeplication_locations` parameters? This would avoid the case of these two parameters getting out of sync.  See comment below about one of the edge cases.

Reply:
I've removed the field and use the georeplication_locations field to determine if it is enabled or not.